### DISCUSSION TITLE
discuss adding private captions in watch anime

### DISCUSSION THREAD

**Discussion by tachib-akiko** [anime-vsub/desktop-web#45] [Q&A]
Shall we make it look like a notepad or integrate it into the progress bar? @tachibana-shin

**Comment by tachibana-shin** [comment]
> Shall we make it look like a notepad or integrate it into the progress bar? @tachibana-shin

this is hard. Now I'm wondering if it's really necessary. we already have `playlist` so this feature is almost superfluous
